template/hooks/common/functions.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
If we're going to be opinionated about this, I think we shouldn't put it in the template. It should be in a blt subdir, under vendor/acquia/blt. This will allow us to roll out new functions and updates.